api/src/services/ik/userProgress/userProgress.ts

```
@@ -0,0 +1,26 @@
+import { db } from 'src/lib/db'
+
+export const deleteUserProgress = async () => {
+  if (!context.currentUser || !context.currentUser.roles.includes('ADMIN')) {
+    return { success: false }
+  }
```

Not a huge deal, but in this case let's throw an forbidden error instead of returning false

`throw new ForbiddenError('Not permitted to call this function.')`